Gemfile.lock

```diff
@@ -0,0 +1,220 @@
+GEM
```

Comment:
We can remove the lockfile. We want to anyway use the latest version of fastlane